styles/styles.css

```diff
@@ -1,39 +1,28 @@
-@import url('https://fonts.googleapis.com/css2?family=Cormorant+Garamond:wght@600&family=Press+Start+2P&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Cormorant+Garamond:wght@600&display=swap');
 
 * {
     font-family: 'Cormorant Garamond', serif;
+    font-size: 16px;
```

Comment:
good try but this is not the correct selector for the root font size 👀

Reply:
got there evenutally....